### PR TITLE
KEYCLOAK-16511 Update app-profile-jee-html5 QS

### PR DESCRIPTION
**Pull request by mhajas**
- load the Keycloak JS adapter from the Keycloak server (If we want to have keycloak.js hardcoded in code I can revert)
- using chrome headless in testsuite
- replace aud with azp in test (This was changed here: https://issues.redhat.com/browse/KEYCLOAK-8483)

### CODE REVIEW

**Comment by mhajas** [comment]
I tested it also with GHA and it is passing now. I needed to do a lot of stabilization there. I am not sure I did it correctly, maybe we should rather update test-helper package with some magic.

**Review by vmuzikar — CHANGES_REQUESTED** [review]
@mhajas Sorry for late review. Changes look good to me, nice stuff!

Could you please just update the README with instructions on how to run the tests, similarly to e.g. [here](https://github.com/keycloak/keycloak-quickstarts/blob/latest/user-storage-simple/README.md#integration-test-of-the-quickstart)?

**Comment by mhajas** [comment]
@vmuzikar done

**Review by vmuzikar — APPROVED** [review]
@mhajas Thanks, LGTM now!

@abstractj Could you please merge this?

File: app-profile-jee-html5/src/test/java/org/keycloak/quickstart/ArquillianProfileJeeHtml5Test.java [hunk 1]
```diff
@@ -122,18 +121,26 @@ public static void cleanUp() throws IOException{
     }
 
     @Before
-    public void setup() {
+    public void setup() throws InterruptedException {
         webDriver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
```
Comment (vmuzikar) [inline app-profile-jee-html5/src/test/java/org/keycloak/quickstart/ArquillianProfileJeeHtml5Test.java:125]: @mhajas During further GH Actions testing I've found a test instability. Could you please add the following line which should fix it?
```java
webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
```

**Review by vmuzikar — CHANGES_REQUESTED** [review]

**Comment by abstractj** [comment]
@mhajas once we managed to get the changes suggested by @vmuzikar, this PR is good to merge.

**Review by vmuzikar — APPROVED** [review]
@mhajas Thanks for the update!

@abstractj Could you please merge?

**Review by abstractj — APPROVED** [review]

**Pull request merged**